packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -268,7 +268,9 @@ class CheckboxListTile extends StatelessWidget {
     this.controlAffinity = ListTileControlAffinity.platform,
     this.autofocus = false,
     this.contentPadding,
-  }) : assert(value != null),
+    this.tristate = false,
```

Comment:
Can you please update the constructor documentation to describe that `tristate` can not be null, and that `value` can be null if `tristate` is true?

Reply:
Done. BTW the prohibition of null value for `isTreeLine`, `selected`, `controlAffinity` and `autofocus` parameters is also missing. But I think this PR is not the right place to fix that.